db/migrate/20150617114935_create_email_requested_receipts.rb

```diff
@@ -0,0 +1,32 @@
+class CreateEmailRequestedReceipts < ActiveRecord::Migration
+  def change
+    create_table :email_requested_receipts do |t|
+      t.references :petition, index: true, foreign_key: true
```

Comment:
It hadn't even crossed my mind - I keep forgetting we're using a fun db.  Maybe something to refactor towards if we add lots more emails?